### ISSUE TITLE
Alert manager not showing alerts on live site

### ISSUE THREAD

**Issue by xmedr** [Metro-Records/la-metro-councilmatic#1036]
The alerts appear on the site after a delay, but they don't appear in the manager for deletion

**Comment by xmedr** [comment]
@hancush I'm having trouble reproducing this locally.

Since our thought is that caching is what's causing issues, I've changed my cache backend to `django.core.cache.backends.db.DatabaseCache` and ran `createcachetable`. I ran [a test in the shell](https://stackoverflow.com/questions/29707274/how-do-you-confirm-django-is-using-memcached/29709199#29709199) and confirmed it was working. Adding and removing alerts from here poses no issues yet, alerts get removed and displayed accordingly which isn't what I expected.

I then tried to see what would happen if I added a `cache.clear()` command after an alert gets [submitted](https://github.com/Metro-Records/la-metro-councilmatic/blob/d5c077d78cae394ec715638607409d91f0bf496d/lametro/views.py#L112) or [deleted](https://github.com/Metro-Records/la-metro-councilmatic/blob/d5c077d78cae394ec715638607409d91f0bf496d/lametro/views.py#L1042) in the views. From here, deleting an alert doesn't remove it from the top, but clears all previous alerts from the manager:
![image](https://github.com/Metro-Records/la-metro-councilmatic/assets/114717958/ebb02122-befc-470f-b1be-7eb43db066a8)

This is more in line with what Shelly's issue looked like before, but this is with the addition of `cache.clear()`. I feel like caching could still be the culprit, but maybe I'm not enabling it correctly?
